controllers/categories.controller.js

```diff
@@ -54,11 +54,12 @@ module.exports = {
 
       const response = await Category.findByPk(id);
 
-      if (!response) throw new ErrorObject('ID not found', 409);
+      if (!response) throw new ErrorObject('ID not found', 422);
```

Comment:
agregarle un mensaje de que no se realizo ningun cambio, como en otro mensaje, creo que era como `ID not found or nothing to change`

Reply:
Y mejor poner Categorie not found, más customizado, como el de transactions